### PR TITLE
[#11878] Merge master into feature

### PR DESCRIPTION
**Pull request by ziqing26**
Part of #11878 

**Outline of Solution**

 

There are two affected PRs:
1. #12811 - merged with resolved conflicts, but need double check on the correctness
2. #12780 - since the conflicts are bulky, might want to port over the logic to the new account request design.

### CODE REVIEW

File: src/client/java/teammates/client/scripts/testdataconversion/DataStoreToSqlConverter.java [hunk 1]
```diff
@@ -57,8 +57,7 @@ public class DataStoreToSqlConverter {
     private UuidGenerator accounRequestUuidGenerator = new UuidGenerator(initialAccountRequestNumber, uuidPrefix);
     private UuidGenerator sectionUuidGenerator = new UuidGenerator(initialSectionNumber, uuidPrefix);
     private UuidGenerator teamUuidGenerator = new UuidGenerator(initialTeamNumber, uuidPrefix);
-    private UuidGenerator deadlineExtensionUuidGenerator = new UuidGenerator(initialDeadlineExtensionNumber,
-            uuidPrefix);
+    private UuidGenerator deadlineExtensionUuidGenerator = new UuidGenerator(initialDeadlineExtensionNumber, uuidPrefix);
```
Comment (ziqing26) [inline src/client/java/teammates/client/scripts/testdataconversion/DataStoreToSqlConverter.java:60]: Here are some of my formatter problems that we did not catch last time. This reverts the formatter change to be the same as on master

**Comment by ziqing26** [comment]
@domoberzin pls help check if changes in #12811 are correctly adapted to our feature changes 🙏

**Review by jayasting98 — APPROVED** [review]
It seems okay to me.

**Review by domoberzin — APPROVED** [review]
LGTM as well

**Pull request merged**